### PR TITLE
Add OAuth SIG meeting link to README

### LINKED ISSUE #32384: Add link of OAuth SIG meeting to README
### Description

We usually announce the location of OAuth SIG Zoom meetings on the OAuth SIG Slack channel, but it is a little difficult for newcomers to find these great meetings.
Also, this change addresses the discussion below.
https://github.com/keycloak/keycloak/discussions/30843

### Discussion

https://github.com/keycloak/keycloak/discussions/30843

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by y-tabata**
Closes https://github.com/keycloak/keycloak/issues/32384

### CODE REVIEW

**Review by francis-pouatcha — APPROVED** [review]

**Comment by tnorimat** [comment]
@abstractj Hello, what do you think about the PR for being aligned with CNCF project

**Comment by abstractj** [comment]
> @abstractj Hello, what do you think about the PR for being aligned with CNCF project

What exactly do you mean @tnorimat ?

Reading the proposal, I don't see any harm about having it in the README file, but I'm curious about what our @keycloak/maintainers think.

Another good alternative is the approach used by the Knative team, providing a shared calendar - https://knative.dev/docs/community/#meetings. That would make things easy to add to the website (I believe).

**Comment by y-tabata** [comment]
@abstractj 
From a CNCF perspective, this proposal stems from [this discussion](https://github.com/keycloak/keycloak/discussions/30843) created by Chris, the CNCF CTO.
Considering Keycloak becoming a CNCF Graduated project, he said that one of the effective ways is to up [the CLOMonitor Score](https://clomonitor.io/projects/cncf/keycloak).
This proposal is an effort to pass [the Best Practice Community meeting check](https://clomonitor.io/docs/topics/checks/#community-meeting).

**Comment by y-tabata** [comment]
@abstractj Could you check the above comment and review this PR?

**Review by stianst — CHANGES_REQUESTED** [review]
I don't think this belongs in the README, but rather on the website.

I'd suggest adding a section to https://www.keycloak.org/community (https://github.com/keycloak/keycloak-web/blob/main/pages/community.ftl) that lists the OAuth SIG, and also community meetings.

**Comment by y-tabata** [comment]
@stianst 
Thank you for your feedback. 
Does it make sense to write a link to the community site's meeting section in the "Help and Documentation" section of the README, after posting information about the meetings on the community site?

**Comment by stianst** [comment]
> @stianst Thank you for your feedback. Does it make sense to write a link to the community site's meeting section in the "Help and Documentation" section of the README, after posting information about the meetings on the community site?

Not sure, doesn't feel the most natural place for it. Maybe it would make more sense in the contributing section? We could definitively expand a bit on that on how people can engage by joining SIGs, finding things to work on, etc.

**Comment by y-tabata** [comment]
Okay, so I'll add the information to the community site and then create a PR to add a link to it in the README.

**Comment by y-tabata** [comment]
@stianst 
I added the link to the community site in the README.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ To write tests, refer to the [writing tests](docs/tests-development.md) guide.
 
 Before contributing to Keycloak, please read our [contributing guidelines](CONTRIBUTING.md). Participation in the Keycloak project is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
 
+Our [website](https://www.keycloak.org/community) lists the community meetings, you may find things to work on by joining them.
```
Comment (stianst) [inline README.md:65]:
```suggestion
Joining a [community meeting](https://www.keycloak.org/community) is a great way to get involved and help shape the future of Keycloak.
```

Comment (y-tabata) [reply README.md]: @stianst 
Thanks, I've incorporated your suggested fixes.

**Review by stianst — APPROVED** [review]

**Pull request merged**